tests/phpunit/rule_test.php

```diff
@@ -1193,4 +1195,28 @@ public function test_blocks_display_after_attempt_finished() {
         $this->assertEquals('secure', $PAGE->pagelayout);
         $this->assertFalse($property->getValue($PAGE->blocks));
     }
+
+    /**
+     * Test we can decide if need to redirect to SEB config link.
+     */
+    public function test_should_redirect_to_seb_config_link() {
+        $this->setAdminUser();
+        $this->quiz = $this->create_test_quiz($this->course, settings_provider::USE_SEB_CONFIG_MANUALLY);
+
+        $reflection = new \ReflectionClass('quizaccess_seb');
+        $method = $reflection->getMethod('should_redirect_to_seb_config_link');
+        $method->setAccessible(true);
+
+        set_config('autoreconfigureseb', '0', 'quizaccess_seb');
+        $_SERVER['HTTP_USER_AGENT'] = 'NOT SEB';
```

Comment:
Is this correct here? As is_using_seb calls strpos($_SERVER['HTTP_USER_AGENT'], 'SEB')

Reply:
good catch

Reply:
fixed